svg/global_attributes.json

```diff
@@ -760,12 +760,12 @@
           "spec_url": "https://www.w3.org/TR/SVG11/struct.html#RequiredFeaturesAttribute",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
https://mdn-bcd-collector.gooborg.com/tests/svg/global_attributes/requiredFeatures also seems invalid, there is no  `requiredFeatures` IDL attribute.